app/views/partials/personal_key/_key.slim

```diff
@@ -7,7 +7,7 @@
     = t('users.recovery_code.header')
   .navy.monospace
     - code.split(' ').each do |word|
-      p.bold.mb0.h3 data-recovery='word'
+      p.bold.mb0.h3(data-recovery="#{word}")
```

Comment:
yep. Additionally, it appears to be a non-standard accessor introduced by Microsoft (didn't know that!). Reference: http://stackoverflow.com/a/19032002

We can use node.textContent, although this is not supported by IE < 9, vs getAttribute which is supported down to IE 8